apps/src/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon.story.tsx

```diff
@@ -37,11 +44,22 @@ DefaultFontAwesomeV6Icon.args = {
   title: 'check',
 };
 
-export const ClassExampleFontAwesomeV6Icon = SingleTemplate.bind({});
-ClassExampleFontAwesomeV6Icon.args = {
-  iconStyle: 'light',
-  iconName: 'brands fa-microsoft',
-  title: 'microsoft',
+export const ClassNameBrandsIconExampleFontAwesomeV6Icon =
+  MultipleTemplate.bind({});
+ClassNameBrandsIconExampleFontAwesomeV6Icon.args = {
+  components: [
+    {
+      iconStyle: 'light',
+      iconName: 'microsoft',
+      className: 'fa-brands',
+      title: 'microsoft',
+    },
+    {
+      iconStyle: 'light',
```

Comment:
Should we delete this example (the other approach seems more intuitive?)

Reply:
Yeah, not sure what's better here since we now already have usage of second example in live code and technically both examples do work 🤔

Reply:
Yeah fair -- FWIW I'd vote that the examples in Storybook should represent how you **should** do it, not how you **can** do it.